src/frontend/packages/lib_classroom/src/components/DashboardClassroom/index.spec.tsx

```diff
@@ -304,7 +303,7 @@ describe('<DashboardClassroom />', () => {
     expect(fetchMock.calls()[0]![0]).toEqual('/api/classrooms/1/');
     expect(
       screen.getByText(
-        `You have joined the classroom as ${token.user?.user_fullname}.`,
+        `You have joined the classroom as ${token.user?.user_fullname || ''}.`,
```

Comment:
In fact here no variable should be used be exactly what is expected by setting the user_fullname in the call of `ltiInstructorTokenMockFactory`